src/SkiddleRequest.php

```diff
@@ -14,27 +14,71 @@ class SkiddleRequest
 {
 
     /**
-     * @var The version of the API we are currently using
+     * @var string The version of the API we are currently using
      */
     const API_VER = "v1";
 
     /**
-     * @var The URL we want to make calls to
+     * @var string The URL we want to make calls to
      */
     const API_URL = 'https://www.skiddle.com/api/';
 
     /**
-     * @var Whether we are in dev mode or not
+     * @var string The URL we want to make calls to (new api)
+     */
+    const API_V3_URL = 'https://api.skiddle.com/v3';
+
+    /**
+     * @var bool Whether we are in dev mode or not
      */
     public $dev_mode;
 
+    /**
+     * @var int Switch between different api versions
+     */
+    public $apiVersion = 1;
+
     /**
      * Basic constructor
      */
     public function __construct()
     {
     }
 
+    public function setApiVersion(int $apiVersion): void
```

Comment:
This package supports PHP 7.0 and up, but `void` return type is only supported in PHP7.1. To maintain existing support, can we remove this?

```suggestion
    public function setApiVersion(int $apiVersion)
```